include/internal/libspdm_common_lib.h

```diff
@@ -150,6 +165,10 @@ typedef struct {
     /* multi-key negotiated result */
     bool multi_key_conn_req;
     bool multi_key_conn_rsp;
+
+    /*responder key pair info*/
+    libspdm_key_pair_info_t key_pair_info[SPDM_MAX_SLOT_COUNT];
```

Comment:
I think we need a new MACRO - `LIBSPDM_MAX_KEY_PAIR_COUNT`

Reply:
Thanks. I have added the new MACRO.